### PR TITLE
feat(serialized-state-manager): create package

### PR DESCRIPTION
**Pull request by KingHenne**

### CODE REVIEW

File: packages/serialized-state-manager/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# @feature-hub/serialized-state-manager
+
+[![Package Version][package-badge]][package-npm]
+[![Website][website-badge]][website] [![API][api-badge]][api]
+
+A Feature Service to manage handing over state to the client when server-side
```
Comment (clebert) [inline packages/serialized-state-manager/README.md]: Alternative proposal for the description:
`A Feature Service for managing the state transfer to the client when rendering on the server.`

File: packages/serialized-state-manager/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@feature-hub/serialized-state-manager",
+  "version": "0.0.0",
+  "description": "A Feature Service to manage handing over state to the client when server-side rendering.",
```
Comment (clebert) [inline packages/serialized-state-manager/package.json]: See comment above.

**Comment by clebert** [comment]
Also please add a link to the website sidebar.

File: packages/serialized-state-manager/README.md (hunk 1 above)
Comment (KingHenne) [reply packages/serialized-state-manager/README.md]: I like it. 👍

File: packages/serialized-state-manager/package.json (hunk 2 above)
Comment (KingHenne) [reply packages/serialized-state-manager/package.json]: > Also please add a link to the website sidebar.

I suggest we add the link when there’s more than empty methods that throw when called. 🙃

**Comment by KingHenne** [comment]
> Also please add a link to the website sidebar.

I suggest we add the link when there’s more than empty methods that throw when called. 🙃

**Pull request merged**